templates/feed.html

```diff
@@ -107,7 +107,7 @@
 {{ post.ListIt(i,3) }}
 </post-text><br>
 </button>
-
+<!-- What does this section do? -->
```

Comment:
this is just a random css/js/html styling thing I chose at the verrrryyy beginning of creating the feed. Basically this creates an "accordion" container for each request. All that information in the html (like class=accordion etc...) is so we can reference this info later in CSS and js to create a nice button/style situation. But again, I'm really not attached to the accordion style, I just chose it in the beginning because it made it so all the info wasn't piling up after another